examples/common/common.rs

```diff
@@ -0,0 +1,57 @@
+use anyhow::{anyhow, Result};
+use reqwest::{blocking::Response, StatusCode};
+use std::{
+    fs::File,
+    io::{self, Write},
+    path::{Path, PathBuf},
+};
+
+use serde::Serialize;
+
+pub fn write_example_sdp_file(port: u16) -> Result<String> {
```

Comment:
I think it would be nice if this functions documentation described the contents of the SDP file in plain language, e.g. `The SDP file will describe an RDP session on localhost with H264 encoding`